config/config.go

```diff
@@ -97,6 +97,12 @@ const (
 	WalletUnlockPasswordFile = "WALLET_UNLOCK_PASSWORD_FILE"
 	// NoOperatorTlsKey is used to start the daemon without using TLS for the operator service
 	NoOperatorTlsKey = "NO_OPERATOR_TLS"
+	// ConnectUrlHost ectUrlHost is the host of the tdex-daemon service to connect to
+	//used by tdexconnect to create connection url
+	ConnectUrlHost = "CONNECT_URL_HOST"
+	// ConnectUrlProto is the http/https protocol of the tdex-daemon service to connect to
+	//used by tdexconnect to create connection url
+	ConnectUrlProto = "CONNECT_URL_PROTO"
```

Comment:
we should set the default here === `127.0.0.1:` + OPERATOR_LISTENING_PORT

Reply:
I do think it's ok to not have a default for `CONNECT_URL_HOST`. When the daemon builds the address for the connect URL, it default's to server one unless this env var has been set.

Reply:
ok, it will be something like `:9000` then, since we do not know the hostname

Reply:
If clients see a host like `:9000` it's kind of standard to assume is localhost, just saying we can eventually do this a tdexdconnect pkg level?